packages/now-next/src/create-serverless-config.ts

```diff
@@ -1,13 +1,16 @@
-import path from 'path';
 import fs from 'fs-extra';
+import path from 'path';
+import semver from 'semver';
+
+import { ExperimentalTraceVersion } from './utils';
 
-function getCustomData(importName: string) {
+function getCustomData(importName: string, target: string) {
   return `
 module.exports = function(...args) {
   let original = require('./${importName}');
 
   const finalConfig = {};
-  const target = { target: 'serverless' };
+  const target = { target: '${target}' };
```

Comment:
```suggestion
  const target = { target };
```

This should already be a string so the template isn't really needed

Reply:
Do note that this is returning a string, and `target` wouldn't be in scope.

Reply:
If anything, we'd probably make this:

```suggestion
  const target = { target: ${JSON.stringify(target)} };
```

Reply:
Seems fine, missed the entire block was a string